### PR TITLE
Allkindsofthingsthatgivejohananxity

### PR DESCRIPTION
**Pull request by WilleVirtanen**
Added config
added itemRepo
added PC Class repo
added PC class choice in the game code

### CODE REVIEW

**Comment by WilleVirtanen** [comment]
i also added that leveling up increases all stats by one!

File: src/Game.java
Comment (Tan-Thai) [inline src/Game.java:1]: Could a potential merge of the setupUser and playerCharacterClassSelection be a logical solution?

Comment (Tan-Thai) [inline src/Game.java:1]: A potential merge between setupUser and playerCharacterClassSelect could be made, thoughts?

**Review by Tan-Thai — APPROVED** [review]
Looks good to me, added comment under the classselection method inside Game.Java.
Do try to name at least the title into a summary, since it will be hard to trace back if we need to find something specific down the line.

**Pull request merged**